CHANGELOG.md

```diff
@@ -5,7 +5,8 @@ Please file changes under `Added`, `Changed`, `Deprecated`, `Removed`, `Fixed` o
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 ## Unreleased
-
+### fixed
```

Comment:
You should already put this under the released header, and give it a new version. Because you are merging it into master